spec/schema/core/option/random.js

```diff
@@ -3,7 +3,7 @@ const seedrandom = require('seedrandom');
 module.exports = {
     register: function(jsf) {
         return jsf.option({
-            random: seedrandom('some seed')
+            random: seedrandom('some seed', {state: true})
```

Comment:
Why you added `state: true`?

I just checked a reference for `seedrandom` but still I don't get it.